src/coreclr/vm/stubmgr.cpp

```diff
@@ -1285,19 +1285,12 @@ BOOL StubLinkStubManager::DoTraceStub(PCODE stubStartAddress,
         LOG_TRACE_DESTINATION(trace, stubStartAddress, "StubLinkStubManager(InstantiatingMethod)::DoTraceStub");
         return TRUE;
     }
-    else if (stub->IsManagedThunk())
+    else if (stub->IsShuffleThunk())
     {
         trace->InitForManagerPush(stubStartAddress, this);
         LOG_TRACE_DESTINATION(trace, stubStartAddress, "StubLinkStubManager(ManagedThunk)::DoTraceStub");
         return TRUE;
     }
-    else if (stub->GetPatchOffset() != 0)
-    {
-        // The patch offset is currently only non-zero in x86 non-IL delegate scenarios.
-        trace->InitForFramePush((PCODE)stub->GetPatchAddress());
-        LOG_TRACE_DESTINATION(trace, stubStartAddress, "StubLinkStubManager::DoTraceStub");
-        return TRUE;
-    }
 
     LOG((LF_CORDB, LL_INFO10000, "StubLinkStubManager::DoTraceStub: patch offset is 0!\n"));
```

Comment:
Add assert to make sure this is true? And add the offset to the log for debugging purposes?

Reply:
Patch offset does not exist after this change. I am just going to update the message.